src/components/controller/ContactInfo/index.js

```diff
@@ -1,5 +1,5 @@
 import { connect } from 'react-redux';
-import Contacts from 'components/view/Contacts';
+import ContactInfo from 'components/view/ContactInfo';
 
 const mapStateToProps = (state) => {
     return {
```

Comment:
I don't think we need `contacts` in the ContactInfo controller, right? Since we're focusing on just one contact

Reply:
Correct